### PR TITLE
fix: authentication error UI and reroute home page

### PR DESCRIPTION
**Pull request by hanlinc27**
## Notion ticket link

[Auth Error UI
](https://www.notion.so/uwblueprintexecs/Authentication-Error-UIs-fdad717a3406406f8006a129ea61b8f8)[Prelim Email Template Styling
](https://www.notion.so/uwblueprintexecs/Prelim-Style-Email-Verification-29fbf67933c94aaeba12aa0acb1f7706)[Reroute Home
](https://www.notion.so/uwblueprintexecs/Route-to-Dashboard-on-Login-dbf3ae54aa874412a68e6184265a6bfb)[Auth Desktop Friendly](https://www.notion.so/uwblueprintexecs/Authentication-Desktop-Friendly-0bbd4c6353f043849e3c1e8fb18df35d)

## Implementation description
* Error UI for signup 
* updating desktop designs
* updating routes on login
Screen Shot 2021-11-29 at 3 20 57 AM
Screen Shot 2021-11-29 at 3 22 39 AM

Screen Shot 2021-11-29 at 3 23 05 AM

## What should reviewers focus on?
* Signup for user is throwing me an error at the moment
* Lint for backend shows no errors locally not sure why CI is breaking

### CODE REVIEW

File: backend/typescript/services/implementations/authService.ts [hunk 1]
```diff
@@ -119,10 +119,13 @@ class AuthService implements IAuthService {
...
+      Thank you for getting involved with mutual aid through Community Fridge KW. We’re thrilled to have you. 
+
       <br><br>
-      We have received a password reset request for your account.
-      Please click the following link to reset it.
+Please confirm that this is the email address (${email}) you would like to use to schedule donations with Community Fridge KW by clicking “Verify”.
+
       <strong>This link is only valid for 1 hour.</strong>
       <br><br>
       <a href=${resetLink}>Reset Password</a>`;
```
Comment (Shehryar21) [inline backend/typescript/services/implementations/authService.ts:131]: Isnt this the reset password email? The content talks about verifying email address.

File: frontend/src/components/pages/Dashboard/index.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const Dashboard = (): JSX.Element => {
   }, [authenticatedUser]);
 
   if (!authenticatedUser) {
-    return <Redirect to={Routes.HOME_PAGE} />;
+    return <Redirect to={Routes.DASHBOARD_PAGE} />;
```
Comment (Shehryar21) [inline frontend/src/components/pages/Dashboard/index.tsx:41]: I believe this should route to the login page

**Review by Shehryar21 — CHANGES_REQUESTED** [review]
The authservice file and test file throws prettier error on vscode as well which is why linting is failing. If you run `yarn fix` in `backend/typescript` folder or run the command in read me, it should fix it.

File: backend/typescript/services/implementations/authService.ts (hunk 1 above)
Comment (hanlinc27) [reply backend/typescript/services/implementations/authService.ts:131]: it was the styling for an updated verifying email template but we don't have to include it for mvp

**Review by jossmurphy — APPROVED** [review]

**Comment by Shehryar21** [comment]
image
I am getting error on sign up saying that the password has to be atleast 6 characters even though the password meets all the requirements.

**Pull request closed without merging**